permissions-policy/resources/permissions-policy-orientation-event.sub.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<body>
+<script src="/orientation-event/resources/orientation-event-helpers.js"></script>
+<script src="orientation-event.js"></script>
+<script>
+"use strict";
+
+Promise.resolve().then(async () => {
+  const name = await waitForOrientationEvent('{{GET[eventName]}}');
+  window.parent.postMessage({ type: "availability-result", name }, "*");
+});
```

Comment:
The Promise.resolve() doesn't seem to add much... Maybe just: 

```suggestion
waitForOrientationEvent('{{GET[eventName]}}').then(name => {
  window.parent.postMessage({ type: "availability-result", name }, "*");
});
```

Reply:
The Promise.resolve() thing was copied from another file in `resources/`. It's not very intuitive, but I think it's equivalent to what other files do by running the code in `window.load`.

I've switched to adding a "load" event handler and replaced async/await with a regular Promise.then() flow like you suggested.